python/lsst/ts/wep/task/cutOutDonutsBase.py

```diff
@@ -724,6 +736,12 @@ def cutOutStamps(self, exposure, donutCatalog, defocalType, cameraName):
             isEffective.append(eff)
             stampsEntropy.append(entro)
 
+            # Store donut radius in the metadata
+            xLeft, xRight, radius, _ = self.fitDonutRadius.fit_radius(image=donutStamp.stamp_im.image.array)
```

Comment:
I think you should move this down below and just use the `run` method on all the stamps at once.